### PR TITLE
Changes to the OSX installer script

### PR DESCRIPTION
**Pull request by Flowdeeps**
Added root password to MYSQL calls
Added cp for htaccess after installation

### CODE REVIEW

File: scripts/osx/newscoop_osx_install.sh [hunk 1]
```diff
@@ -85,6 +85,9 @@ chmod 775 plugins install cache images public conf log
 # default admin / password
 composer self-update
 composer install --prefer-dist
-sudo ./application/console newscoop:install --fix --database_name newscoop --database_user root --no-client
+sudo ./application/console newscoop:install --fix --database_name newscoop --database_user root --database_password root --no-client
 sudo php upgrade.php
 sudo php $SCRIPTPATH/../../newscoop/scripts/fixer.php
+
+# create an apache htaccess file after the installation process
+cp $SCRIPTPATH/htaccess.dist $SCRIPTPATH/.htaccess
```
Comment (ahilles107) [inline scripts/osx/newscoop_osx_install.sh]: file should be copied by installer.

Comment (Flowdeeps) [inline scripts/osx/newscoop_osx_install.sh]: Didn't seem to when running it for the first time on a clean system for me.

Maybe Mark's experience varies. @thnkloud9 ?

Comment (ahilles107) [inline scripts/osx/newscoop_osx_install.sh:93]: any news?

Comment (thnkloud9) [inline scripts/osx/newscoop_osx_install.sh:93]: I know this is the worst thing to say, but it worked on my machine without this line.  .htaccess was copied automatically after running the installer.  However, no idea why it didn't work for Flowdeeps, but I can confirm that it wasn't there after successfully running this script.

Comment (Flowdeeps) [inline scripts/osx/newscoop_osx_install.sh:93]: I'm working on installing on exotic systems at the moment. I'll revisit this and see if I can fix it properly in a bit.

**Pull request closed without merging**